<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Barcode</class>
 <widget class="QMainWindow" name="Barcode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Barcode Mode</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="groupPeminjam">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="lAnggotaPeminjam">
        <property name="styleSheet">
         <string notr="true">color: rgb(48, 48, 48);
font-size: 18px;
font-weight: bold;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bBarcodeSelesai">
        <property name="text">
         <string>F5. Selesai</string>
        </property>
        <property name="shortcut">
         <string>F5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="bBarcodeTutup">
        <property name="text">
         <string>F6. Tutup</string>
        </property>
        <property name="shortcut">
         <string>F6</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLineEdit" name="lScanAnggota">
      <property name="placeholderText">
       <string>Scan / Input No Anggota</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabBarcode">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabBarcodePinjam">
       <attribute name="title">
        <string>Peminjaman</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lScanBuku">
            <property name="placeholderText">
             <string>Scan Buku</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tblPeminjaman">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
